org.mixedrealitytoolkit.spatialmanipulation/ObjectManipulator/ObjectManipulator.cs

```diff
@@ -531,6 +544,10 @@ protected struct LogicImplementation
 
         private bool wasKinematic = false;
 
+        private IManipulationSpecialization activeSpecialization = null;
+
+        private bool specializedManipulationActive => activeSpecialization != null;
```

Comment:
nit: PascalCase on properties

I realize this link is for MRTK2, but we weren't able to get around to publishing the coding guidelines for MRTK3 before...you know... 🙃 I'm hoping we'll adopt something similar here though, as these have worked for this project in the past: https://learn.microsoft.com/windows/mixed-reality/mrtk-unity/mrtk2/contributing/coding-guidelines?view=mrtkunity-2022-05#naming-conventions